packages/server-wallet/src/models/channel.ts

```diff
@@ -152,7 +152,14 @@ export class Channel extends Model implements ChannelColumns {
   };
 
   static jsonAttributes = ['vars', 'participants', 'initialSupport'];
-
+  static async forIds(channelIds: Bytes32[], txOrKnex: TransactionOrKnex): Promise<Channel[]> {
```

Comment:
This method is added in this PR, but it is not used elsewhere. Is there a reason it's included here?

Reply:
resolved in efa6f61